### PR TITLE
Combine openshift_node and openshift_node_upgrade

### PR DESCRIPTION
**Pull request by michaelgugino**
Currently, having openshift_node and openshift_node_upgrade
as two distinct roles has created a duplication across
handlers, templates, and some tasks.

This commit combines the roles to reduce duplication
and bugs encountered by not putting code in both places.

### CODE REVIEW

File: roles/openshift_node/meta/main.yml [hunk 1]
```diff
@@ -13,9 +13,15 @@ galaxy_info:
   - cloud
 dependencies:
 - role: openshift_node_facts
+  when: not (openshift_node_upgrade_in_progress | default(False))
```
Comment (michaelgugino) [inline roles/openshift_node/meta/main.yml]: I'm not crazy about adding booleans here, but it seems like the least-effort way to combine these roles.

After the roles are combined, I propose we refactor the meta depends out of both.

Comment (kwoodson) [reply roles/openshift_node/meta/main.yml]: Agreed.

**Comment by mtnbikenc** [comment]
A fix was merged for install failures
/test install

**Pull request merged**